application/backend/src/main/resources/apps/typerefinery/web_root/light.css

```diff
@@ -1,12 +1,11 @@
 :root {
-    --primary-background-color: #FFF;
+    --primary-panel-color: #FFF;
```

Comment:
primary-bg-color for the page backgrounds
card-bg-color for the components background color
border-color